src/main/java/org/apache/sling/scripting/sightly/render/RuntimeObjectModel.java

```diff
@@ -95,6 +96,14 @@
      */
     Date toDate(Object object);
 
+    /**
+     * Convert the given object to a {@link Instant} object
```

Comment:
`s/to a/to an/` :)

Reply:
fixed the c&p typo in https://github.com/apache/sling-org-apache-sling-scripting-sightly-runtime/pull/7/commits/026497d9d0d1eba7ca4a95c629d532331d68f0b3